.babelrc

```diff
@@ -1 +1,14 @@
-{ "presets": ["env", "flow"] }
+{
+  "presets": [
+    ["@babel/preset-env", { "modules": false }],
+    "@babel/preset-flow"
+  ],
+  "env": {
+    "cjs": {
+      "presets": [["@babel/preset-env", { "targets": { "node": 6 } }]]
```

Comment:
this can be bumped up to 10, the current LTS version

Reply:
Bumped. Should I also add `engines` entry to `package.json`?

Reply:
Yes please

Reply:
Added, please check